Rakefile

```diff
@@ -33,4 +34,4 @@ namespace :test do
       Rake::Task['test:exercise'].execute(exercise: exercise)
     end
   end
-end
+end
```

Comment:
Tried using Github's merge conflict resolution web interface.
Won't be trying that again. :disappointed: